src/docker/mod.rs

```diff
@@ -136,13 +136,24 @@ impl<I: NodeImage> Node<I> {
                 }
             })
     }
+
+    pub fn stop_remove(&self) -> impl Future<Item = (), Error = ()> {
```

Comment:
❓   Do you actually need this? With `Drop` implemented on `Node<T>` and the nodes now gracefully going out of scope at the end of `start_env`, they should be dropped implicitly.

If not then just call this function on the `Drop` impl to reduce duplication.

Reply:
@luckysori yet they are not stopped when doing ctrl-c on create-comit-app...

https://www.reddit.com/r/rust/comments/aqkdo4/dropping_on_shutdown/